2016-12-11 15:52:13 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Dashboard could not be found.
Exception Attributes: array (
  'class' => 'Dashboard',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /dashboard/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:34:07 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\src\Controller\PagesController.php line=47 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:34:07 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:34:07 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:34:07 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:34:07 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Style.css could not be found.
Exception Attributes: array (
  'class' => 'Style.css',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /style.css
Referer URL: http://test.my.local/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(67): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(93): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:34:07 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Log could not be found.
Exception Attributes: array (
  'class' => 'Log',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /log/script.js
Referer URL: http://test.my.local/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(93): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:34:08 Error: [Cake\Routing\Exception\MissingControllerException] Controller class W could not be found.
Exception Attributes: array (
  'class' => 'W',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /w/php/filegetcontents.php
Referer URL: http://test.my.local/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(93): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:39:54 Warning: Warning (2): file_get_contents(): php_network_getaddresses: getaddrinfo failed: ���̂悤�ȃz�X�g�͕s���ł��B  in [D:\xampp\htdocs\work\test\src\Controller\PagesController.php, line 46]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
file_get_contents - [internal], line ??
App\Controller\PagesController::display() - APP/Controller\PagesController.php, line 46
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 435
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 122
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 96
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 83
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 62
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 81
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 92
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:39:54 Warning: Warning (2): file_get_contents(https://tabelog.com/osaka/A2701/A270401/27086999/) [<a href='http://php.net/function.file-get-contents'>function.file-get-contents</a>]: failed to open stream: php_network_getaddresses: getaddrinfo failed: ���̂悤�ȃz�X�g�͕s���ł��B  in [D:\xampp\htdocs\work\test\src\Controller\PagesController.php, line 46]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
file_get_contents - [internal], line ??
App\Controller\PagesController::display() - APP/Controller\PagesController.php, line 46
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 435
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 122
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 96
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 83
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 62
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 81
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 92
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:39:54 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:39:54 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:39:54 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:39:54 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:40:10 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\src\Controller\PagesController.php line=47 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:40:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:40:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:40:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\PagesController.php:47) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 12:40:11 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Advertise could not be found.
Exception Attributes: array (
  'class' => 'Advertise',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /advertise/bundle?restaurant_id=27086999&type=rstdtl
Referer URL: http://test.my.local/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:40:11 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Contents could not be found.
Exception Attributes: array (
  'class' => 'Contents',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /contents/rstdtl_side?restaurant_id=27086999&user_id=&user_type=nologin&_=1483965611289
Referer URL: http://test.my.local/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 12:40:57 Warning: Warning (2): file_get_contents(https://tabelog.com/osaka/A2701/A270401/27086999/): failed to open stream: �ڑ��ς݂̌Ăяo���悪���̎��Ԃ�߂��Ă������������Ȃ��������߁A�ڑ��ł��܂���ł����B�܂��͐ڑ��ς݂̃z�X�g���������Ȃ��������߁A�m�����ꂽ�ڑ��͎��s���܂����B
 in [D:\xampp\htdocs\work\test\src\Controller\PagesController.php, line 46]
Request URL: /
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
file_get_contents - [internal], line ??
App\Controller\PagesController::display() - APP/Controller\PagesController.php, line 46
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 435
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 122
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 96
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 83
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 62
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 81
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 92
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:21:40 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\search.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\search.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('search')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('search', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(59): Cake\Controller\Controller->render('search')
#3 [internal function]: App\Controller\SearchController->search('search')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-09 13:22:52 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\search.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\search.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('search')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('search', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(59): Cake\Controller\Controller->render('search')
#3 [internal function]: App\Controller\SearchController->search('search')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-09 13:22:53 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\search.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\search.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('search')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('search', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(59): Cake\Controller\Controller->render('search')
#3 [internal function]: App\Controller\SearchController->search('search')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-09 13:23:14 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\search.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\search.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('search')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('search', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(59): Cake\Controller\Controller->render('search')
#3 [internal function]: App\Controller\SearchController->search('search')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-09 13:23:47 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\search.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\search.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('search')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('search', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(59): Cake\Controller\Controller->render('search')
#3 [internal function]: App\Controller\SearchController->search('search')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-09 13:24:16 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\src\Controller\SearchController.php line=39 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:24:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:39) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:24:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:39) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:24:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:39) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:24:17 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Contents could not be found.
Exception Attributes: array (
  'class' => 'Contents',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /contents/rstdtl_side?restaurant_id=27086999&user_id=&user_type=nologin&_=1483968257064
Referer URL: http://test.my.local/search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 13:24:17 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Advertise could not be found.
Exception Attributes: array (
  'class' => 'Advertise',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /advertise/bundle?restaurant_id=27086999&type=rstdtl
Referer URL: http://test.my.local/search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 13:32:15 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\src\Controller\SearchController.php line=44 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:32:15 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:44) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:32:15 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:44) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:32:15 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\src\Controller\SearchController.php:44) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-09 13:32:16 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Advertise could not be found.
Exception Attributes: array (
  'class' => 'Advertise',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /advertise/bundle?restaurant_id=27086999&type=rstdtl
Referer URL: http://test.my.local/search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-09 13:32:16 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Contents could not be found.
Exception Attributes: array (
  'class' => 'Contents',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /contents/rstdtl_side?restaurant_id=27086999&user_id=&user_type=nologin&_=1483968736191
Referer URL: http://test.my.local/search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-22 07:45:03 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Site could not be found.
Exception Attributes: array (
  'class' => 'Site',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /site
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-22 07:45:43 Warning: Warning (2): file_get_contents(): php_network_getaddresses: getaddrinfo failed: ���̂悤�ȃz�X�g�͕s���ł��B  in [D:\xampp\htdocs\work\test\src\Controller\SearchController.php, line 38]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
file_get_contents - [internal], line ??
App\Controller\SearchController::search() - APP/Controller\SearchController.php, line 38
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 435
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 122
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 96
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 83
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 62
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 81
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 92
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:45:43 Warning: Warning (2): file_get_contents(https://tabelog.com/osaka/A2701/A270401/27086999/) [<a href='http://php.net/function.file-get-contents'>function.file-get-contents</a>]: failed to open stream: php_network_getaddresses: getaddrinfo failed: ���̂悤�ȃz�X�g�͕s���ł��B  in [D:\xampp\htdocs\work\test\src\Controller\SearchController.php, line 38]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
file_get_contents - [internal], line ??
App\Controller\SearchController::search() - APP/Controller\SearchController.php, line 38
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 435
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 122
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 96
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 83
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 62
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 81
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 92
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:45:43 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:45:43 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:45:43 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:45:43 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-22 07:47:21 Error: [Cake\View\Exception\MissingTemplateException] Template file "Js\js.ctp" is missing.
Exception Attributes: array (
  'file' => 'Js\\js.ctp',
)
Request URL: /js/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('js')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('js', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\JsController.php(54): Cake\Controller\Controller->render('js')
#3 [internal function]: App\Controller\JsController->js('js')
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\JsController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-28 14:16:25 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::index() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'index',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-29 14:52:37 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::s() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 's',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/s
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-29 14:52:41 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::inde() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'inde',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/inde
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-29 14:53:16 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:53:16 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:36 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:53:36 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:36 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:36 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:36 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:46 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/index/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:53:46 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/index/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:46 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/index/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:46 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/index/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:46 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/index/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:53:57 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::a() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'a',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/a/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-29 14:54:06 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:54:06 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:54:06 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:54:06 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:54:06 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:54:49 Error: [Cake\View\Exception\MissingTemplateException] Template file ".ctp" is missing.
Exception Attributes: array (
  'file' => '.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('/')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('/', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(62): Cake\Controller\Controller->render('/')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:55:01 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(64): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:55:01 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:01 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:01 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:01 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:10 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:55:10 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:55:10 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:56:57 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:56:57 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:56:57 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:56:57 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:56:57 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:16 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:57:16 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:16 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:22 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-29 14:57:22 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:22 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:22 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-29 14:57:22 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:15:42 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::search() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'search',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-31 15:15:49 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->search()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-31 15:15:49 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:15:49 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:15:49 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:15:49 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:16:08 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::index() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'index',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-31 15:16:10 Error: [Cake\Controller\Exception\MissingActionException] Action SearchController::index() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'SearchController',
  'action' => 'index',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#2 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2017-01-31 15:16:25 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-31 15:16:25 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:16:25 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:16:25 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:16:25 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:17:18 Error: [Cake\View\Exception\MissingTemplateException] Template file "Search\.ctp" is missing.
Exception Attributes: array (
  'file' => 'Search\\.ctp',
)
Request URL: /search/
Stack Trace:
#0 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName('')
#1 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render('', NULL)
#2 D:\xampp\htdocs\work\test\src\Controller\SearchController.php(63): Cake\Controller\Controller->render('')
#3 [internal function]: App\Controller\SearchController->index()
#4 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#5 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#6 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\SearchController))
#7 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#9 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#10 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#11 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#13 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#15 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 D:\xampp\htdocs\work\test\webroot\index.php(37): Cake\Http\Server->run()
#18 {main}


2017-01-31 15:17:18 Warning: Warning (512): Unable to emit headers. Headers sent in file=D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php line=753 in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 48]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 48
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:17:18 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 146]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitStatusLine() - CORE\src\Http\ResponseEmitter.php, line 146
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 54
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:17:18 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



2017-01-31 15:17:18 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Error\Debugger.php:753) in [D:\xampp\htdocs\work\test\vendor\cakephp\cakephp\src\Http\ResponseEmitter.php, line 173]
Request URL: /search/
Client IP: 127.0.0.1
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 146
header - [internal], line ??
Cake\Http\ResponseEmitter::emitHeaders() - CORE\src\Http\ResponseEmitter.php, line 173
Cake\Http\ResponseEmitter::emit() - CORE\src\Http\ResponseEmitter.php, line 55
Cake\Http\Server::emit() - CORE\src\Http\Server.php, line 118
[main] - ROOT\webroot\index.php, line 37



